tests/nimdoc/invalidrst.nim

```diff
@@ -0,0 +1,4 @@
+proc foo*() =
```

Comment:
convention is to name helper modules not directly compiled with mXXX, eg `minvalidrst.nim`